frontend/cypress/pages/RegisterPage.js

```diff
@@ -1,13 +1,13 @@
 class RegisterPage{
 
-    register(register){
-      cy.visit('/register')
-      cy.get('[data-testid="email-register"]').type(register.email);
-      cy.get('[data-testid="password-register"]').type(register.password);
-      cy.get(`[data-testid="${register.roles}"]`,{ timeout: 10000 }).click();
-      cy.get('[data-testid="register-btn"]').click();
-      cy.get('div[class="Toastify__toast-body"]').should("have.text",register.alert_msg); 
-    }
+  register(register){
+    cy.visit('/register')
+    cy.get('[data-testid="email-register"]').type(register.email);
+    cy.get('[data-testid="password-register"]').type(register.password);
+    cy.get('[data-testid="rolesRegister"]',{ timeout: 20000 }).contains(register.roles).click();
+    cy.get('[data-testid="register-btn"]').click();
+    cy.get('div[class="Toastify__toast-body"]').should("have.text",register.alert_msg); 
```

Comment:
Just one more thing, the last instruction for each test should be an expectation.

Reply:
@edimossilva in Cypress, the `.should` method works like an expectation about the current subject, so in this case the test is expecting the element to have the given text.

Reply:
My bad!

@andersonmalheiro , In other stacks I am used to using only one matcher per project. 
e.g. I can configure lint for using  `expect` or `should` for every scenario. Do you know what is the convention for cypress? 

In the previous code, we have `expect`, should we use both? can you check what does the style guide says?

Reply:
From what I understand from the examples, `expect` is mostly used when we have a value || function || object (like an endpoint response) and we need to test something with it, like in unit testing. On the other hand, `should` is more suitable when we need to test something on the UI elements of the page. 

I could be wrong, but that's what I understood from seeing examples from the documentation and on github 😅.

Reply:
LGTM! 
Lets go with it!